SonarQube.Client/Api/DefaultConfiguration.cs

```diff
@@ -50,7 +50,8 @@ public static RequestFactory Configure(RequestFactory requestFactory)
                 .RegisterRequest<IGetRoslynExportProfileRequest, V6_60.GetRoslynExportProfileRequest>("6.6")
                 .RegisterRequest<IGetOrganizationsRequest, V7_00.GetOrganizationsRequest>("7.0")
                 .RegisterRequest<IGetIssuesRequest, V7_20.GetIssuesRequestWrapper>("7.2")
-                .RegisterRequest<IGetHotspotRequest, V8_6.GetHotspotRequest>("8.6");
+                .RegisterRequest<IGetHotspotRequest, V8_6.GetHotspotRequest>("8.6")
+                .RegisterRequest<IGetTaintVulnerabilitiesRequest, V7_20.GetTaintVulnerabilitiesRequest>("7.2");
```

Comment:
Won't this mean that the "view taint vulnerabilities" feature will work against older versions of SonarQube from v7.2 onwards, and if so is it what we want?

From a technical point of view it probably doesn't make any difference, but it makes this more complicated from a marketing and support point of view. Probably worth checking what the PMs view is.

Reply:
Good point, I asked Marco. For now he suggests 7.2, he will let us know if this changes.